source/administration/plugins.rst

```diff
@@ -20,6 +20,20 @@ The Plugin Marketplace is a collection of plugins that can greatly increase the
 
 .. image:: https://user-images.githubusercontent.com/915956/66891467-1b18eb80-ef9e-11e9-9de3-37a3c5899bd8.png
 
+Plugin Labels
+~~~~~~~~~~~~~~~~
+
+Plugins in the Marketplace are labeled to make it easier for administrators to choose plugins that fit their company's security and risk policies if they do not allow for community plugins to be used. 
+
+**Mattermost Official Plugins**
```

Comment:
In v5.20, at least, we're not labelling official plugins at all, given the "spam" this label would produce. Should we invert the language here instead?